### PR TITLE
added mvp of rules modal

### PR DESCRIPTION
**Pull request by sylfort**
Added the written rules, will think about where to put the rules button in the UI next.

### CODE REVIEW

File: src/components/component/rules.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                <li>
+                  Failure to Steal: If another player issued the same cat card,
+                  you fail to steal and cannot take the cheese cards.
+                </li>
+                <li>
+                  Reusing Cat Cards: You can put the cat card back into your
+                  hand and use it again.
+                </li>
+                <li>
+                  Game End: The game ends after the eight round when the deck
```
Comment (iamzaidmohammed) [inline src/components/component/rules.jsx:100]: I think here should be eighth instead of eight

**Comment by iamzaidmohammed** [comment]
Everything looks alright but I don't see the changes I made for the quitGame logic in your code.

**Review by dominic-source — APPROVED** [review]
I have reviewed the code and it is awesome.

**Comment by dominic-source** [comment]
> Everything looks alright but I don't see the changes I made for the quitGame logic in your code.

Since the changes are in a separate file, we may import the component into the main component that renders our code.
We may also need to create routes for different pages.

**Pull request merged**